classes/Cart.php

```diff
@@ -1501,6 +1505,13 @@ public function updateQty(
 
     /**
      * Customization management.
+     *
+     * @param mixed $quantity
```

Comment:
Multiline:

Shouldn't `$quantity` be `int` or something like `int|null`?
Shouldn't `$id_customization` be `int` or something like `int|null`?
Shouldn't `$id_product` be `int` or something like `int|null`?
Shouldn't `$id_product_attribute` be `int` or something like `int|null`?
Shouldn't `$id_address_delivery` be `int` or something like `int|null`?
Shouldn't `$operator` be `string`?

Reply:
side note: this is my first review, I did not understand making a multiline comment

Reply:
@jolelievre can you confirm, then I make a script for it?

Reply:
in this case:
```php
    @param int $quantity
    @param mixed $id_customization
    @param int $id_product
    @param int $id_product_attribute
    @param int $id_address_delivery
    @param string $operator
```

But how do you plan to script this, there is no general rule For example even `$id_customization` here can be empty (probable a string), so how do you compute the type?

Reply:
@jolelievre I can't script it then.